src/components/index.js

```diff
@@ -10,7 +10,7 @@ export { COLORS } from './Colors'
 export { EmailInput } from './EmailInput'
 export { Faq } from './Faq'
 export { Form } from './Form'
-export { Grid, Row, Column, useGridSorting } from './Grid'
+export { Grid, Pagination, Row, Column, useGridSorting } from './Grid'
```

Comment:
is this an unused import?

Reply:
It’s an export that is now needed now that I wanted to consume it from outside of EDS.